### ISSUE TITLE
How to Import lottie-web for Angularjs when using bower?

### ISSUE THREAD

**Issue by SagarStake** [airbnb/lottie-web#2008]
I am using angularjs with bower, I am trying to include lottie-web in the angular app, but can not import the library into angular. Is there any example to demonstrate how to use this library in angularjs? 
PS: I have tried bodymovin as well as lottie-web

**Comment by FavyTeam** [comment]
yes, lottie package or lottie directive is not declare in angular.js. If someone create one, share it to the world. It will benefit.
